### PR TITLE
Add more entries on .gitignore

### PR DESCRIPTION
**Pull request by enidgjoleka**
This PR adds one more entry on .gitignore:
* ebin/

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
As per https://github.com/uwiger/parse_trans/pull/44#issuecomment-774344538:

"
> .idea -> needed for developers who work intelliJ Idea

Shouldn't developers who use this, include the exception in their own `.gitignore_global`? To me, `.gitignore` was always about files that the application itself might generate, e.g. `logs`, `_build`, `rebar3.crashdump`, ...
"

I'd apply the same reasoning as with the `.DS_Store`, for example (generated in Mac).

(imagine we add exceptions for all editors...)

Also, `ebin/` is for `rebar` 2. Still valid to use, but not the one `erlang-certifi` aims for.

**Comment by enidgjoleka** [comment]
I will remove the `.idea` entry. 

As I mentioned on the other PR: https://github.com/uwiger/parse_trans/pull/44#issuecomment-774920967
the system I am working on is quite old and does not make use of rebar3 yet.
Hence, we need to deal with `ebin`.

**Comment by paulo-ferraz-oliveira** [comment]
> Hence, we need to deal with ebin.

Sure, `ebin` is reasonable, which is why I wrote "Also, ebin/ is for rebar 2. Still valid to use, but not the one erlang-certifi aims for.".

**Comment by enidgjoleka** [comment]
Hi @paulo-ferraz-oliveira 
I was wondering there is anything left in this PR that might stop it from being merged.
If so please let me know and I would be happy to work on it.

Thanks.

**Comment by paulo-ferraz-oliveira** [comment]
Hi, @enidgjoleka. I don't think there is anything missing, apart from the approval and merging by @benoitc.

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,4 @@ _plugins
 _tdeps
 logs
 /_build
+ebin/
```
Comment (benoitc) [inline .gitignore:19]: can you add an empty line at the end of the file?

Comment (enidgjoleka) [reply .gitignore]: Fixed

**Comment by benoitc** [comment]
merged. Sorry for the delay.

**Pull request merged**